src/configure/configure.ts

```diff
@@ -154,6 +157,23 @@ class Orchestrator {
         }
     }
 
+    private async getGithubPatToken(): Promise<void> {
+        const redirectHelper = new RedirectLinkHelper();
+        await redirectHelper.loadAll();
+        if (this.inputs.sourceRepository.repositoryProvider === "github" || redirectHelper.repoAnalysisServiceFramework === ServiceFramework.Moda) {
+            this.inputs.githubPATToken = await this.controlProvider.showInputBox(constants.GitHubPat, {
+                placeHolder: Messages.enterGitHubPat,
+                prompt: Messages.githubPatTokenHelpMessage,
+                validateInput: (inputValue) => {
+                    return !inputValue ? Messages.githubPatTokenErrorMessage : null;
+                }
+            });
+            if (!!this.inputs.githubPATToken) {
+                this.inputs.githubPATToken = this.inputs.githubPATToken.toLowerCase();
```

Comment:
why the case change? we should trust user to do right thing. On the other side it can break the PAT if github decides to issue pat with capital case letters.

Reply:
Github right now doesn't support PAT with capital letters. So this is required. Agreed, it might break if Github changes to capital letters, but that's very unlikely as a lot of systems depending on GitHub pat will break

Reply:
But why take this extra burden which, even though unlikely, has a chance to break.
If user is inputting a wrong cased PAT he will end up in failure. Lets not try to fix user's error.

Reply:
Alright